src/main/java/com/suse/saltstack/netapi/client/impl/HttpClientConnection.java

```diff
@@ -1,25 +1,34 @@
 package com.suse.saltstack.netapi.client.impl;
 
-import com.suse.saltstack.netapi.client.Connection;
-import com.suse.saltstack.netapi.config.ClientConfig;
-import static com.suse.saltstack.netapi.config.ClientConfig.*;
-import com.suse.saltstack.netapi.exception.SaltStackException;
-import com.suse.saltstack.netapi.parser.JsonParser;
+import static com.suse.saltstack.netapi.config.ClientConfig.PROXY_HOSTNAME;
```

Comment:
Nitpick: looks like you re-organized imports automatically here, with a different style than the one that was used before. Please make sure that your patches only change lines that are supposed to be changed - in general you should not reorganize imports for code that's not being changed by your contribution.

Exception of course is if you are submitting a restyling PR :-)